runtime/bcutil/ROMClassWriter.cpp

```diff
@@ -651,6 +651,11 @@ ROMClassWriter::writeFields(Cursor *cursor, bool markAndCountOnly)
 			if (iterator.hasTypeAnnotation()) {
 				modifiers |= J9FieldFlagHasTypeAnnotations;
 			}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+			if (iterator.isNullRestricted()) {
+				modifiers |= J9FieldFlagIsNullRestricted;
```

Comment:
I don't see this flag been used and checked. I guess they will be added in the next PR ?

Reply:
Yes, I intend to use this for the putfield/putstatic NPE's and remaining class verification checks.